plainly-plugin/src/ui/types/project.ts

```diff
@@ -10,6 +10,7 @@ export interface Project extends ModelBaseWithDates {
   };
   revisionHistory?: ProjectRevision[];
   analyzed: boolean;
+  size: number;
```

Comment:
i don't feel size has any real relevance here.. I would like more to see if we have any templates, maybe creation or modification date.. Let's discuss